build.sbt

```diff
@@ -47,6 +46,22 @@ lazy val root = (project in file("."))
       Resolver.jcenterRepo
     ),
     libraryDependencies ++= compileDeps ++ testDeps("test") ++ testDeps("it"),
+    dependencyOverrides ++= Set(
+      "org.eclipse.jetty" % "jetty-server" % jettyVersion % "it",
+      "org.eclipse.jetty" % "jetty-servlet" % jettyVersion % "it",
+      "org.eclipse.jetty" % "jetty-security" % jettyVersion % "it",
```

Comment:
Is all this override stuff needed. Not sure why we need to be on exactly that jettyVersion. Service still worked when I removed it.

Reply:
It's needed if we uplift to the newest version of Scalatest-plus, that introduced non-trivial exceptions though. I've left it in as it will be needed in the future.